public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+    <p>Web development is a broad term for the work involved in developing 
+    a web site for the Internet (World Wide Web) or an intranet (a private network). 
+    Web development can range from developing the simplest static single page of 
+    plain text to the most complex web-based internet applications, electronic businesses, and social network services.
+    A more comprehensive list of tasks to which web development commonly refers, 
+    may include web engineering, web design, web content development, client liaison, client-side/server-side scripting, 
+    web server and network security configuration, and e-commerce development. 
+    Among web professionals, "web development" usually refers to the main non-design aspects of building web sites: writing markup and coding. 
+    Most recently Web development has come to mean the creation of content management systems or CMS. 
+    These CMS can be made from <em>scratch, proprietary or open source.</em> 
+    In broad terms the CMS acts as middleware between the database and the user through the browser. 
+    A principle benefit of a CMS is that it allows <strong>non-technical people to make changes to their web site without having technical knowledge.</strong></p>
+</body>
+</html>
```

Comment:
Your code looks great. You did a great job with the challenges for lesson 2. You are making great progress, and for these being the first assignments, you did amazing.